beacon-chain/rpc/service.go

```diff
@@ -557,8 +425,6 @@ func (s *Service) Start() {
 	// Register reflection service on gRPC server.
 	reflection.Register(s.grpcServer)
 
-	validatorServer.PruneBlobsBundleCacheRoutine()
```

Comment:
why is this moved ?

Reply:
it's not, it's part of the Start() function still but the validator server needed to be set somewhere, i moved all of the initialization functions out of start and introduced the validator server as another variable to be used.

see line 587 s.validatorServer.PruneBlobsBundleCacheRoutine()